R/pred_se_wrap.R

```diff
@@ -32,7 +33,8 @@
 pred_se_wrap <- function(df_trans, var_interest, model,
                          type = 'levels', base_rn = 1,
                          at_var_interest = NULL,
-                         vcov_mat = NULL){
+                         vcov_mat = NULL,
+                         dof = model$df.residual){
```

Comment:
We're in dependency hell a little bit with these interior functions. What is the norm here for what these things should default to? Since the defaults are not the same as the default of the function that calls them. I just worry.